reports/emulator_machineinfo.go

```diff
@@ -0,0 +1,26 @@
+package reports
+
+import (
+	"log"
+)
+
+func EmulateMachineInfo() (MachineInfo, error) {
+
+	win32OS, err := GetWin32OS()
+	if err != nil {
+		// TODO return the error here?
+		log.Printf("reports: getting win32 os: %s", err)
+	}
+
+	report := MachineInfo{
+		os_vers: win32OS.Caption,
+	}
+
+	return report, nil
+}
+
+// os_vers is what sal expects - TODO change sal's behavior?
+// https://github.com/salopensource/sal/blob/master/server/views.py#L1939
+type MachineInfo struct {
+	os_vers string
```

Comment:
Is this being encoded to JSON?

I suggest you change the go struct to

```
type MachineInfo struct {
    OSVersion string `json:"os_vers"`
}
```

This will encode the go structure into the correct JSON format.